fetch.js

```diff
@@ -204,6 +206,21 @@
 
         if (this._bodyBlob) {
           return Promise.resolve(this._bodyBlob)
+        } else if (this.body) {
+          var reader = this.body.getReader();
+          var chunks = [];
+          var pump = function(){
```

Comment:
Minor: we usually put a space in between `)` and `{` (same applies to next line)